docs/getting_started.md

```diff
@@ -0,0 +1,325 @@
+# Getting started
+
+## Data generation process
+
+The data generation process uses this analogy: generated data flows from source to sink.
+
+To generate data is then necessary to define:
+
+* `source`: what data is generated, eg. data model
+* `sink`: where data is sent to, eg. ES index
+* `flow`: how data is transmitted, eg. how fast or how much?
+* `schema`: fields definition, eg. ECS 8.2.0
+
+Each of the above is handled by its own REST API endpoint. An arbitray
+number of sources, sinks, flows and schemas can be defined on the same
+server.
+
+## Install
+
+Currently Geneve is packaged only for [Homebrew](https://brew.sh), you
+need first to install the Geneve tap
+
+```shell
+$ brew tap elastic/geneve
+```
+
+then the tool itself
+
+```shell
+$ brew install geneve
+```
+
+## REST API server
+
+Data is generated by the Geneve server, you start it with
+
+```shell
+$ geneve serve
+2023/01/31 16:40:23 Control: http://localhost:9256
+```
+
+The server keeps the terminal busy with its logs, to stop just press `^C`.
+The first line in the log shows where to reach it, this is the base url of
+the server, all the API endpoints are reachable (but not browseable) under
+`api/`.
+
+For the rest of this document we'll assume that the following shell
+variables are set:
+
+* `$GENEVE` points to the Geneve server, url `http://localhost:9256`
+* `$TARGET_ES` is the url of the target Elasticsearch instance
+* `$TARGET_KIBANA` is the corresponding Kibana's url
+
+Now open a separate terminal to operate on the server with curl.
+
+## Loading the schema
+
+The schema describes the fields that can be present in a generated
+document. At the moment it needs to be explicitly loaded into the server.
+
+Download the latest version (or any other, if you have preferences) from
+https://github.com/elastic/ecs/releases and search for file `ecs_flat.yml`
+in the folder `ecs-X.Y.Z/generated/ecs/`.
```

Comment:
nit: Helpful dev-ex here would be to have a command/route that would take an ECS version (or default to latest with no input) and then download locally from the ecs releases and then load.

This sorta thing is really helpful when having to reproduce bugs on an older stack version, and having to manually fetch/load the specific schema adds friction here.

Reply:
Here I'm not following you. What are you suggesting to do here?

Reply:
If you just pass it a version, like `v8.6.0`, then the Geneve can call out to https://github.com/elastic/ecs/releases, download the compressed file and extract the ecs_flat.yml.  Icing on the cake would be to also automatically load it into Geneve with an appropriate name.

Reply:
I think it would be a nice enhancement but the documentation is pretty clear on what the user needs to do to get the schema.